addons/whats-happening-filter/addon.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "What's happening filter",
+  "description": "Lets you select which types of messages to hide or show in \"What's Happening?\" box.",
+  "info": [
+    {
+      "type": "notice",
+      "text": "To open settings for this addon, click the triangle in \"What's Happening?\" box.",
```

Comment:
Is there a reason why it can't be on the settings page?

Reply:
There's no proper way to change the settings from a usescript at the moment. This is similar to "message filters" addon.

Reply:
Yes. You can read more info about this here: https://github.com/ScratchAddons/ScratchAddons/issues/4124#issuecomment-1133609197